Packs/MicrosoftTeams/ReleaseNotes/1_4_70.md

```diff
@@ -0,0 +1,7 @@
+
+
+#### Integrations
+
+##### Microsoft Teams via Webhook
+
+- Adjust the integration to the new microsoft teams webhook format- using workflows instead of incoming webhook.
```

Comment:
```suggestion
- Update the integration to adapt to the new format of Microsoft Teams webhook by using workflows instead of the incoming webhook. for more information- <link to documentation>.
```